libraries/SRV_Channel/SRV_Channels.cpp

```diff
@@ -102,6 +105,20 @@ const AP_Param::GroupInfo SRV_Channels::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO_FRAME("_AUTO_TRIM",  17, SRV_Channels, auto_trim, 0, AP_PARAM_FRAME_PLANE),
     
+    // @Param: _CANSRV_BM
+    // @DisplayName: RC Out channels to be transmitted as servo over CAN bus
+    // @Description: Bitmask with one set for channel to be transmitted as a servo command over CAN bus
+    // @Bitmask: 0: Servo 1, 1: Servo 2, 2: Servo 3, 3: Servo 4, 4: Servo 5, 5: Servo 6, 6: Servo 7, 7: Servo 8, 8: Servo 9, 9: Servo 10, 10: Servo 11, 11: Servo 12, 12: Servo 13, 13: Servo 14, 14: Servo 15
+    // @User: Standard
+    AP_GROUPINFO("_CANSRV_BM", 18, SRV_Channels, can_servo_bm, 0),
+
+    // @Param: _CANESC_BM
+    // @DisplayName: RC Out channels to be transmitted as ESC over CAN bus
+    // @Description: Bitmask with one set for channel to be transmitted as a ESC command over CAN bus
+    // @Bitmask: 0: ESC 1, 1: ESC 2, 2: ESC 3, 3: ESC 4, 4: ESC 5, 5: ESC 6, 6: ESC 7, 7: ESC 8, 8: ESC 9, 9: ESC 10, 10: ESC 11, 11: ESC 12, 12: ESC 13, 13: ESC 14, 14: ESC 15, 15: ESC 16, 16: ESC 17, 17: ESC 18, 18: ESC 19, 19: ESC 20
```

Comment:
Here too, I would have it as Channel. Also, we should limit this to 16 as that is the maximum number of channels we have in this library.